src/parquet/column/properties.h

```diff
@@ -78,10 +78,12 @@ class PARQUET_EXPORT ReaderProperties {
 ReaderProperties PARQUET_EXPORT default_reader_properties();
 
 static int64_t DEFAULT_PAGE_SIZE = 64 * 1024 * 1024;
+static bool DEFAULT_IS_DICTIONARY_ENABLED = true;
```

Comment:
Per discussions elsewhere and to be consistent with other Parquet implementations this seems like the right approach (and using whatever value encoding is set as the fallback encoding)